plugins/rocksdb/src/test/java/org/hyperledger/besu/plugin/services/storage/rocksdb/unsegmented/RocksDBColumnarKeyValueStorageTest.java

```diff
@@ -299,7 +300,7 @@ private SegmentedKeyValueStorage<RocksDbSegmentIdentifier> createSegmentedStore(
       final Path path,
       final List<SegmentIdentifier> segments,
       final List<SegmentIdentifier> ignorableSegments) {
-    return new RocksDBColumnarKeyValueStorage(
+    return new OptimisticRocksDBColumnarKeyValueStorage(
```

Comment:
Could we do similar test with TransactionDb / PessimisticRocksDBColumnarKeyValueStorage.

Reply:
Extended the current class to create and use TransactionDB and OtimisticTransactionDB. Thanks for the suggestion.